tutorials/quick_start/pytorch_fw/quant.py

```diff
@@ -39,6 +41,26 @@ def get_tpc(target_platform_name: str, target_platform_version: str) -> TargetPl
     return mct.get_target_platform_capabilities('pytorch', target_platform_name, target_platform_version)
 
 
+def get_target_kpi(model, weights_compression, representative_data_gen, core_config, tpc):
```

Comment:
All comments regarding text, arguments names and documentation are relevant here as well